common/djangoapps/student/migrations/0050_auto__add_manualenrollmentaudit__chg_field_userprofile_bio.py

```diff
@@ -0,0 +1,224 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding model 'ManualEnrollmentAudit'
+        db.create_table('student_manualenrollmentaudit', (
+            ('id', self.gf('django.db.models.fields.AutoField')(primary_key=True)),
+            ('enrollment', self.gf('django.db.models.fields.related.ForeignKey')(to=orm['student.CourseEnrollment'], null=True)),
+            ('enrolled_by', self.gf('django.db.models.fields.related.ForeignKey')(to=orm['auth.User'], null=True)),
+            ('enrolled_email', self.gf('django.db.models.fields.CharField')(max_length=255, db_index=True)),
+            ('time_stamp', self.gf('django.db.models.fields.DateTimeField')(auto_now_add=True, null=True, blank=True)),
+            ('state_transition', self.gf('django.db.models.fields.CharField')(max_length=255)),
+            ('reason', self.gf('django.db.models.fields.TextField')(null=True)),
+        ))
+        db.send_create_signal('student', ['ManualEnrollmentAudit'])
+
+
+        # Changing field 'UserProfile.bio'
+        db.alter_column('auth_userprofile', 'bio', self.gf('django.db.models.fields.CharField')(max_length=3000, null=True))
```

Comment:
@muhhshoaib following up on @cahrens comment on migrations, I'm surprised by the addition of the bio column. Seems like we might have picked that up from somewhere else? Can you investigate.